packages/target-tools/src/parsing.js

```diff
@@ -0,0 +1,40 @@
+import { isString } from "./lodash";
+import { parseURI } from "./index";
```

Comment:
@jasonwaters I was having trouble importing `parseURI` from the `parse-uri` module so I had to import it from `index.js` instead. This means that I could not export `parseURL` from `index.js` as I was getting circular reference errors.

Reply:
changing to `parseDomainFull` avoids the need to use `parseURI` so you should be OK